### ISSUE TITLE
Introduce option to disable UI server (ideally both at run and at build time)

### ISSUE THREAD

**Issue by schickling** [DataDog/temporalite#33]
Please introduce option to disable UI server - ideally both at run and at build time which reduces the binary size.

**Comment by schickling** [comment]
Thanks @jlegrone! Would you please provide a quick comment on how this can be done with the recent release?

**Comment by jlegrone** [comment]
Yep! Please see the new docs at https://github.com/DataDog/temporalite#web-ui
